src/supertux/player_status.hpp

```diff
@@ -40,11 +40,13 @@ class PlayerStatus final
   PlayerStatus();
   void reset();
   void add_coins(int count, bool play_sound = true);
+  void take_checkpoint_coins() { coins -= std::max(coins / 10, 25); }
```

Comment:
AppVeyor complains about
```
'max': identifier not found
```

Reply:
It cannot find std::max `error C2039: 'max': is not a member of 'std'`. Maybe it's an unreliable transitive header inclusion.
https://ci.appveyor.com/project/supertux/supertux-o8t59/builds/25955016#L5821